package-lock.json

```diff
@@ -0,0 +1,2905 @@
+{
+  "name": "Django",
+  "lockfileVersion": 3,
```

Comment:
Thanks for notifying that and asking. It depends on the NPM version that I have locally. I'm on `9.4.0`. I think it is safer if we use `lockfileVersion` to `2` instead of 3, I mentioned the reason at the very bottom of this comment.

  1. I researched a bit, the information on `lockfileVersion` is a bit sparse on the NPM documentation, but we can grasp some points for sure here:
https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json

  2. If we are about to build NPM library/package, it is better if we skip the lockfileVersion. But when we use in the application or CI environments, we could stick to the lockfile as it says here:
https://www.twilio.com/blog/lockfiles-nodejs

  3. [Whilst doing this, I also found that it is better if we do `npm ci` instead of `npm install` here](https://github.com/django/django/blob/3bbe22dafcc69c5ffa79707f5a74eb1faf466e12/.github/workflows/schedule_tests.yml#L51)

It is mentioned here in detail on `npm ci`.
https://docs.npmjs.com/cli/v9/commands/npm-ci

So, coming back to `lockfileVersion` - I would suggest that we could do it as `2` because the current NPM version that I find here is `https://www.npmjs.com/package/download` `8.0.0`